src/ontology/uberon-edit.obo

```diff
@@ -224086,6 +224086,14 @@ is_a: UBERON:0001830 ! minor salivary gland
 relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date "2023-06-19T12:41:05Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480066
+name: interscapular region
+def: "The region in the upper part of the back between the shoulder blades." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0000475 ! organism subdivision
```

Comment:
@paulacalipho, consider adding `'part of' some dorsum`.

Reply:
@bvarner-ebi thanks for reviewing.